runtime/avn/src/governance/tracks.rs

```diff
@@ -0,0 +1,142 @@
+const APP_GENERAL_ADMIN: Curve =
+    Curve::make_reciprocal(4, 28, percent(80), percent(50), percent(100));
+const SUP_GENERAL_ADMIN: Curve =
+    Curve::make_reciprocal(7, 28, percent(10), percent(0), percent(50));
+const APP_REFERENDUM_CANCELLER: Curve = Curve::make_linear(17, 28, percent(50), percent(100));
+const SUP_REFERENDUM_CANCELLER: Curve =
+    Curve::make_reciprocal(12, 28, percent(1), percent(0), percent(50));
+const APP_REFERENDUM_KILLER: Curve = Curve::make_linear(17, 28, percent(50), percent(100));
+const SUP_REFERENDUM_KILLER: Curve =
+    Curve::make_reciprocal(12, 28, percent(1), percent(0), percent(50));
+const APP_WHITELISTED_CALLER: Curve =
+    Curve::make_reciprocal(16, 28 * 24, percent(96), percent(50), percent(100));
+const SUP_WHITELISTED_CALLER: Curve =
+    Curve::make_reciprocal(1, 28, percent(20), percent(5), percent(50));
+
+const TRACKS_DATA: [(u16, pallet_referenda::TrackInfo<Balance, BlockNumber>); 5] = [
+    (
+        0,
+        pallet_referenda::TrackInfo {
+            name: "root",
+            max_deciding: 1,
+            decision_deposit: 1000 * AVT,
+            prepare_period: 2 * HOURS,
+            decision_period: 28 * DAYS,
+            confirm_period: 24 * HOURS,
+            min_enactment_period: 24 * HOURS,
+            min_approval: APP_ROOT,
+            min_support: SUP_ROOT,
+        },
+    ),
+    (
+        1,
+        pallet_referenda::TrackInfo {
+            name: "whitelisted_caller",
+            max_deciding: 100,
+            decision_deposit: 1000 * AVT,
+            prepare_period: 30 * MINUTES,
+            decision_period: 28 * DAYS,
+            confirm_period: 10 * MINUTES,
+            min_enactment_period: 10 * MINUTES,
+            min_approval: APP_WHITELISTED_CALLER,
+            min_support: SUP_WHITELISTED_CALLER,
+        },
+    ),
+    (
+        2,
+        pallet_referenda::TrackInfo {
+            name: "general_admin",
+            max_deciding: 10,
+            decision_deposit: 500 * AVT,
+            prepare_period: 2 * HOURS,
+            decision_period: 28 * DAYS,
+            confirm_period: 3 * HOURS,
+            min_enactment_period: 10 * MINUTES,
+            min_approval: APP_GENERAL_ADMIN,
+            min_support: SUP_GENERAL_ADMIN,
+        },
+    ),
+    (
+        3,
+        pallet_referenda::TrackInfo {
+            name: "referendum_canceller",
+            max_deciding: 1000,
+            decision_deposit: 1000 * AVT,
+            prepare_period: 2 * HOURS,
+            decision_period: 7 * DAYS,
+            confirm_period: 3 * HOURS,
+            min_enactment_period: 10 * MINUTES,
+            min_approval: APP_REFERENDUM_CANCELLER,
+            min_support: SUP_REFERENDUM_CANCELLER,
+        },
+    ),
+    (
+        4,
+        pallet_referenda::TrackInfo {
+            name: "referendum_killer",
+            max_deciding: 1000,
+            decision_deposit: 2000 * AVT,
```

Comment:
Have we confirmed this amounts for these operations with the team & business? I assume yes.